src/app/support/containers/support-page/support-page.container.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+
+import { ISupport } from '../../interfaces/support.interface';
+import { SupportService } from '../../services/support.service';
+@Component({
+  selector: 'app-support-page-container',
+  templateUrl: './support-page.container.html',
+})
+export class SupportPageContainer implements OnInit {
+
+  constructor(private _supportService: SupportService) { }
+
+  public ngOnInit(): void {
+  }
+
+  public submit(formData: ISupport): void {
+    this._supportService.send(formData)
+      .subscribe((value) => console.log(value));
```

Comment:
You forgot to unsubscribe from the stream! 
And logs are recommended to delete!